tests/test_rds/test_rds.py

```diff
@@ -1076,6 +1082,29 @@ def test_copy_db_snapshots(delete_db_instance: bool):
     assert result["TagList"] == []
 
 
+@mock_aws
+def test_copy_db_snapshot_invalid_arns():
+    conn = boto3.client("rds", region_name=DEFAULT_REGION)
+
+    invalid_arn = (
+        f"arn:aws:rds:{DEFAULT_REGION}:123456789012:this-is-not-a-snapshot:snapshot-1"
+    )
+    with pytest.raises(ClientError) as ex:
+        conn.copy_db_snapshot(
+            SourceDBSnapshotIdentifier=invalid_arn,
+            TargetDBSnapshotIdentifier="snapshot-2",
+        )
+    assert "is not a valid identifier" in ex.value.response["Error"]["Message"]
+
+    cross_account_arn = f"arn:aws:rds:{DEFAULT_REGION}:112233445566:snapshot:snapshot-1"
+    with pytest.raises(ClientError) as ex:
+        conn.copy_db_snapshot(
+            SourceDBSnapshotIdentifier=cross_account_arn,
+            TargetDBSnapshotIdentifier="snapshot-2",
+        )
+    assert "not yet implemented in moto" in ex.value.response["Error"]["Message"]
```

Comment:
No need to cover something that's not implemented.  And, per my previous comment, if you raise `NotImplementedError` CodeCov will ignore that line in the coverage calculation.